docs/workflow.rst

```diff
@@ -156,13 +165,13 @@ same criteria must be applied to the AST catalog.
 Commands to edit the files, both to remove flagged sources and eliminate sources
 that don't have full imaging coverage, and to create ds9 region files:
 
-.. code-block:: console
+  .. code-block:: console
+
+     $ python -m beast.tools.cut_catalogs phot_catalog_with_sourceden.fits phot_catalog_cut.fits \
+           --partial_overlap --region_file --flagged --flag_filter F475W
 
-   $ python -m beast.tools.cut_catalogs phot_catalog_with_sourceden.fits phot_catalog_cut.fits \
-         --partial_overlap --region_file --flagged --flag_filter F475W
-   $
-   $ python -m beast.tools.cut_catalogs ast_catalog.fits ast_catalog_cut.fits \
-         --partial_overlap --region_file --flagged --flag_filter F475W
+     $ python -m beast.tools.cut_catalogs ast_catalog.fits ast_catalog_cut.fits \
+           --partial_overlap --region_file --flagged --flag_filter F475W
```

Comment:
I can't actually find what changed in this part

Reply:
Just changed the indent so that it matched all the rest of console blocks.